### PR TITLE
Add Upstream/Downstream dropdowns

### PR DESCRIPTION
**Pull request by Dalje-et**
This PR introduces Upstream/downstream dropdown menus at several different positions in the plugin. This is done to align the plugin more with the Instana UI.

### CODE REVIEW

File: src/util/analyze_util.ts [hunk 1]
```diff
@@ -5,13 +5,17 @@ import Granularity from "../types/granularity";
 
 const MAX_ALLOWED_DATA_POINTS = 1000;
 
-export function createTagFilter(filter: TagFilter) {
-  const tagFilter = {
+export function createTagFilter(filter: TagFilter, useEntity = false) {
```
Comment (schmidtl) [inline src/util/analyze_util.ts:8]: where do we call this with true?

Comment (Dalje-et) [reply src/util/analyze_util.ts:8]: We call it with true for "analyze applications". "Analyze websites" does not have the new entity addition

File: src/query_ctrl.ts [hunk 2]
```diff
@@ -24,6 +25,7 @@ export class InstanaQueryCtrl extends QueryCtrl {
   uniqueOperators: Array<Selectable> = operators;
   uniqueBeaconTypes: Array<Selectable> = beaconTypes;
   sloSpecifics: Array<Selectable> = sloInfo;
+  callToEntities: Array<Selectable> = call_to_entities;
```
[the remark this answers is not in the document]
Comment (Dalje-et) [reply src/query_ctrl.ts:28]: `call_to_entities[0]` does not work. I replaced it with `this.callToEntities[0]`

File: src/util/analyze_util.ts (hunk 1 above)
Comment (Dalje-et) [reply src/util/analyze_util.ts:8]: My bad! That as actually an error. I fixed it with my latest commit!

**Pull request merged**